src/Helper/HeadLink.php

```diff
@@ -408,6 +408,8 @@ public function createDataStylesheet(array $args)
         if ($args && is_array($args[0])) {
             $extras = array_shift($args);
             $extras = (array) $extras;
+        } else {
```

Comment:
`else` is not needed here. Move the variable definition before `if` and use `$extras = []`.